tools/base/fpgaconf/fpgaconf.c

```diff
@@ -372,9 +372,13 @@ int parse_args(int argc, char *argv[])
 		fprintf(stderr, "No GBS file\n");
 		return -1;
 	}
-	config.filename = argv[optind];
-
-	return 0;
+	config.filename = canonicalize_file_name(argv[optind]);
+	if (config.filename) {
+		return 0;
+	} else {
+		fprintf(stderr, "Error locating GBS file specified: \"%s\"\n", strerror(errno));
```

Comment:
Consider to use 'print_err' here?

Reply:
`print_err` is specific to fpga_result codes.